pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -711,7 +723,7 @@ stdenv.mkDerivation (finalAttrs: {
     # runtime; otherwise we can't and we need to reboot.
     interfaceVersion = 2;
 
-    inherit withCryptsetup withHostnamed withImportd withLocaled withMachined withPortabled withTimedated withUtmp util-linux kmod kbd;
+    inherit withCryptsetup withHostnamed withImportd withKmod withLocaled withMachined withPortabled withTimedated withUtmp util-linux kmod kbd;
```

Comment:
I am not sure if this is necessary, but it seems that way, because if we go from kmod enable to kmod disabled or vice-versa we might have issues reloading some of the systemd units on `nixos-rebuild switch`. Correct me if I misunderstand this.